cli/cmd/range_slice_test.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestValidateRangeSlice(t *testing.T) {
+	assertNoError(t, validateRangeSlice(nil))
+	assertNoError(t, validateRangeSlice([]string{}))
+	assertNoError(t, validateRangeSlice([]string{"23"}))
+	assertNoError(t, validateRangeSlice([]string{"23-23"}))
+	assertNoError(t, validateRangeSlice([]string{"25-27"}))
+	assertNoError(t, validateRangeSlice([]string{"23"}))
+	assertNoError(t, validateRangeSlice([]string{"23"}))
+
+	assertError(t, validateRangeSlice([]string{""}), "not a valid port")
+	assertError(t, validateRangeSlice([]string{"notanumber"}), "not a valid port")
+	assertError(t, validateRangeSlice([]string{"not-number"}), "not a valid lower-bound")
+	assertError(t, validateRangeSlice([]string{"-23-25"}), "ranges expected as")
+	assertError(t, validateRangeSlice([]string{"-23"}), "not a valid lower-bound")
+	assertError(t, validateRangeSlice([]string{"25-23"}), "upper-bound must be greater than or equal to")
```

Comment:
Really like this extensive testing

Reply:
_Hopefully_ I improved the usability of the tests.  Previously, any failure really didn't provide the context as to which case failed.  Updated for parameterized testing which will indicate the specific test case that caused the failure.